components/Header.jsx

```diff
@@ -1,8 +1,8 @@
 import React from "react";
 
-const Header = ({ title }) => {
+const Header = ({ title, color ="acm-black"}) => {
```

Comment:
don't use a default color, pass in acm black in the places we need it. otherwise default behaviors can mess us up later on